scripts/assistant.ts

```diff
@@ -0,0 +1,53 @@
+import { createReadStream, readFileSync } from 'fs';
+import OpenAi from 'openai';
+import { env } from './env.js';
+
+const openai = new OpenAi({ apiKey: env.OPENAI_API_KEY });
+
+/**
+ * Uploads an array of documents to Open AI and returns their file IDs
+ */
+const uploadDocuments = async (documents: string[]): Promise<OpenAi.Files.FileObject[]> => {
+	if (documents.length >= 20) {
```

Comment:
nit: would be good to make it as constant

Reply:
It is not a value I can decide 😢. OpenAI has a limit of 20 files per assistant.